src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -289,22 +287,43 @@ void Engine::GetPerformanceStreams(Config& config, const std::shared_ptr<ngraph:
         streams = config.streamExecutorConfig._streams == 1 ? 0 : config.streamExecutorConfig._streams;
     }
 
-    const auto latency_name = std::string(CONFIG_VALUE(LATENCY)) + "_" + std::string(ov::num_streams.name());
-    const auto tput_name = std::string(CONFIG_VALUE(THROUGHPUT)) + "_" + std::string(ov::num_streams.name());
-
-#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
-    // TODO: This WA for CI checks will be removed after export/import refactor for MT 2.0 which will be track in CVS-112149
-    streams = 1;
-#endif
-
     get_num_streams(streams, ngraphFunc, config);
 
-    hints_props.insert({latency_name, std::to_string(latency_streams)});
-    hints_props.insert({tput_name, std::to_string(config.streamExecutorConfig._streams)});
-    ngraphFunc->set_rt_info(hints_props, "intel_cpu_hints_config");
     config._config[CONFIG_KEY(CPU_THROUGHPUT_STREAMS)] = std::to_string(config.streamExecutorConfig._streams);
 }
 
+Config Engine::ReadConfigAndGetPerformanceStreams(const std::map<std::string, std::string>& config,
+                                                  const std::shared_ptr<ngraph::Function>& function,
+                                                  bool imported) {
+    // TODO: Clarify the behavior of SetConfig method. Skip eng_config or not?
+    Config conf = engConfig;
+
+    conf.readProperties(config);
+
+    // import config props from caching model
+    if (imported && !is_cpu_map_available()) {
```

Comment:
Didn't get why we read optimal number of streams from imported model only in case legacy threading? I would expect we should do it for both scenarious.

Reply:
Reading optimal number of streams from imported model is needed by the legacy threading, for threading 2.0, streams is recaculated each time and reading streams from imported model  is not needed. And this logic will be removed after legacy threading is removed.

Reply:
Yes, it is recalculated each time. The problem is: `GetPerformanceStreams` uses  `get_model_prefer_threads` to calculate prefered number of stream based on model itself. This function utilizes common `MemBandwidthPressureTolerance` which works only with common OV opset while model imported from cache is already in CPU opset. In other words might return different results for `compile_model` and `import_model` calls. Not sure if this tests reproduces such behavior.

Reply:
In the test case where only performance hint is set, different results of `get_model_prefer_threads` will cause test failure. So far we do not observe different result in CI checks and local test.

Since `model_perfer_threads` is internal parameters which cannot be changed by user application, we can keep it in cache file and reuse it in `import_model` to avoid this risk. Will update.

Reply:
@dmitry-gorokhov model_prefer_threads which is computed by MemBandwidthPressureTolerance has been saved to cache for compile_model and loaded from cache for import_model.